config/initializers/doxxer.rb

```diff
@@ -0,0 +1,12 @@
+class Doxxer
+  BASE_URL = 'https://just-arrived.herokuapp.com'
+  def self.example_for(model_klass)
+    model = ActiveModel::SerializableResource.new(model_klass.new)
+    JSON.pretty_generate(model.serializable_hash)
+  end
+
+  def self.curl_for(name:, id: nil)
+    path = [name, id].compact.join('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.